runtime/hydradx/src/xcm.rs

```diff
@@ -192,7 +199,7 @@ impl pallet_xcm::Config for Runtime {
 	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
 	type XcmRouter = XcmRouter;
 	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
-	type XcmExecuteFilter = Everything;
+	type XcmExecuteFilter = Nothing;
```

Comment:
@apopiak @dmoka this effectively breaks the exchange and send back assets scenario correct?

Reply:
if this is the case i would reconsider this change

Reply:
I would say it makes sense to define the more permissive filter in the context of the ExchangeAsset stuff, not here.